index.js

```diff
@@ -0,0 +1,27 @@
+// const fs = require('fs');
+
+// const sumNumbers = (fileA, fileB) => {
+//     const numberA = parseInt(fs.readFileSync(fileA, 'utf8'));
+//     const numberB = parseInt(fs.readFileSync(fileB, 'utf8'));
+//     const sum = numberA + numberB;
+//     console.log(sum);
+// }
+
+const fs = require('fs/promises');
+
+const sumNumbers = async (fileA, fileB) => {
+    try {
+        const dataA = await fs.readFile(fileA, 'utf8');
```

Comment:
It's also good to use e.g resolve or some another method from path module in this case